regolith/templates/beamplan.txt

```diff
@@ -0,0 +1,77 @@
+\documentclass[prl,aps,tighten,amsmath,amssymb,floatfix]{revtex4-1}
```

Comment:
I think this should be ``beamplan.tex`` not ``beamplan.txt``

Reply:
fixed.